Header header

float64 latitude_deg
float64 longitude_deg

# Cartesian coordinate system
float64 heading_rad